vendor/bat-native-ads/src/bat/ads/internal/base/strings/string_conversions_util.cc

```diff
@@ -18,4 +22,27 @@ std::string BoolToString(const bool value) {
   return value ? kTrue : kFalse;
 }
 
+std::vector<float> ConvertStringToVector(std::string string) {
+  const std::vector<std::string> vector_string = base::SplitString(
+      string, " ", base::TRIM_WHITESPACE, base::SPLIT_WANT_NONEMPTY);
+  std::vector<float> vector;
+  for (const std::string& element_string : vector_string) {
+    double element;
+    base::StringToDouble(element_string, &element);
```

Comment:
We should check the return value. See `base/strings/string_number_conversions.h`